src/dockerflow/django/middleware.py

```diff
@@ -11,6 +12,15 @@
 from . import views
 
 
+def is_authenticated(user):
+    """Check if the user is authenticated but do it in a way that
+    it doesnt' cause a DeprecationWarning in Django >=1.10"""
+    is_authenticated = user.is_authenticated
+    if VERSION < (1, 10):
```

Comment:
Since this code is executed on every request, it makes sense to just store the result of `VERSION < (1, 10)` in a module level variable and use it here, no need to waste cycles doing this comparison on every request.